package.json

```diff
@@ -42,16 +39,19 @@
   },
   "devDependencies": {
     "@babel/node": "^7.20.7",
-    "@jeremyckahn/minami": "^1.3.1",
     "@testing-library/dom": "^8.3.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.0.0",
     "@testing-library/user-event": "^13.2.1",
+    "@types/dinero.js": "^1.9.0",
+    "@types/lodash.sortby": "^4.7.7",
     "@types/markdown-table": "^2.0.0",
     "@types/react": "^17.0.2",
+    "@types/uuid": "^9.0.2",
     "@typescript-eslint/eslint-plugin": "^2.23.0",
     "@typescript-eslint/parser": "^2.23.0",
     "bittorrent-tracker": "^9.19.0",
+    "browserslist": "^4.21.9",
```

Comment:
This was yelling at me while running tests, so I updated the config. This shouldn't affect too much.